### DISCUSSION TITLE
Electric Substations

### DISCUSSION THREAD

**Discussion by celinaproffen** [pypsa-meets-earth/pypsa-earth#281] [Q&A] [answered]
Dear Pypsa Team, 

I am interested in predicting village level (grid!) electrification for villages in Ethiopia. My research has led me to understand that what matters for this type of inference is more the available electric substations that are used to convert the voltage down to usable levels, than the actual power plants. Do you already have data on electric substations in Africa and in Ethiopia specifically?

All the best,
Celina

**Comment by davide-f** [comment, marked as answer]
Dear Celina,

Thank you for your contribution and enrichment of the discussion.

For your specific question regarding the data the answer is yes: our workflow automatically downloads open street map data on network assets, thus you could use the tool to download such data.
Obviously data may not be complete and that's why we are working on assets identification.

Moroever, I strongly believe that that information can be useful to understand levels of demand. The installed power assets available anywhere the world are a measure of demand because if there wasn't enough demand in a given area, then utilities would have no interests in installing such assets (although sometimes the driver for some investments is not necessarily cost efficiency).
That's why your comment is absolutely welcome and may be an interesting info to our demand team, head by @stephenjlee .

However, your point is not obvious, as in my view it is not assets that lead to demand, rather the opposite: since there is enough expected demand, then utilities may decide to install the assets (in practise it is not that simple, though it is an approximation). This is especially true for areas with no electric lines: latent demand is there but cannot be inferred by the availability of network assets.
My believe is that the "density" of power assets can be correlated to higher energy demand, by exploiting the rational "utilities usually do not invest in assets where demand is too low", but it is not obvious whether this correlation may be better justified by other third inputs of the study (examples population density, remoteness, etc.).

If interested, we do have weekly meetings of the demand team every Wednesday at 10PM CET on [Discord](https://discord.gg/AnuJBk23FU); if you could make it we could discuss the above or we can also chat here as well.
If you like, we could have a chat on discord to understand how your work could fit the ongoing activities.

**Reply by celinaproffen** [reply]
Dear Davide, 

This sounds very reasonable. I guess that the focus of my research (inferring if and when an individual village was connected to the grid) is a bit different from yours (seeing which regions are developing an important demand of electricity?). Is that right?

Is there a way to infer where electricity substations are located exactly (GPS) based on your data? 

In all cases, I think this project is amazing, and your team is doing a great job on this platform!

All the best,
Celina

**Reply by pz-max** [reply]
Hi @celinaproffen,
Great to hear about your interest. We are using mainly OSM data which also includes low-voltage substations incl. GPS location! :) The LV data is in general far from complete so it's good to see your research focus.

To extract the data for Ethiopia and see what's going on, you have to run this [script](https://github.com/pypsa-meets-africa/pypsa-africa/blob/main/scripts/download_osm_data.py) with the snakemake workflow and Ethopia as two-letter string in the [config.yaml](https://github.com/pypsa-meets-africa/pypsa-africa/blob/main/config.default.yaml). I can recommend going through the [hackathon material ](https://github.com/pypsa-meets-africa/pypsa-africa-hackathon)to learn how to do this.

As an alternative to the demand meeting, you can also join our data and workflow meeting on Thursday at 1.30PM UK time (it's open and hosted at Discord) so we can better understand what you need. Here are the meeting details: [https://github.com/pypsa-meets-africa/pypsa-africa#get-involved](https://github.com/pypsa-meets-africa/pypsa-africa#get-involved)

**Comment by davide-f** [comment]
As max has said, some data for the substations are already available bit we are also working in inferring where network infrastructure are using AI, see the [detect_energy](https://github.com/pypsa-meets-africa/detect_energy) package lead by @LukasFrankenQ.
However this last activity is ongoing and dependent upon images availability. We always welcome contributions also in terms of data, if you may have some and are interested in collaborating, please reach out

**Comment by stephenjlee** [comment]
Thanks for linking me, @davide-f. I think the OP's question is pretty different from the demand workstream - we use transformer locations where available but have not pursued that data in Ethiopia. I do know Prof Jay Taneja from UMass has very compelling research on village-level electricity access estimation in Kenya. I may suggest you try connecting with his group.
